ceph_iscsi_config/common.py

```diff
@@ -152,6 +153,17 @@ def _get_ceph_config(self):
 
         return cfg_dict
 
+    def init_config(self):
+        ioctx = self.ceph.cluster.open_ioctx(self.pool)
+        try:
+            with rados.WriteOpCtx(ioctx) as op:
+                op.new(1) # try to exclusively create the config object
```

Comment:
Nit: use the constant ```LIBRADOS_CREATE_EXCLUSIVE``` [1]

https://github.com/ceph/ceph/blob/master/src/pybind/rados/rados.pyx#L320